lib/components/regions_map/components/serach_places/view_models/search_places_bloc.dart

```diff
@@ -59,17 +73,32 @@ class SearchPlacesBloc extends Bloc<SearchPlacesEvent, SearchPlacesState> {
   }
 
   Future<void> _onPredictionSelected(OnPredictionSelected event, Emitter<SearchPlacesState> emit) async {
-    final result = await GetPlaceDetailsUseCase().run(GetPlaceDetailsUseCase.input(event.prediction.placeId));
-    if (result.isError) {
-      emit(state.copyWith(pageState: PageState.failure));
-    } else {
-      final PlacesDetailsResponse details = result.asValue!.value;
-      emit(state.copyWith(
-          placeSelected: Place(
-        placeText: event.prediction.description,
-        lng: details.result.geometry.location.lng,
-        lat: details.result.geometry.location.lat,
-      )));
+    bool isRegionSelected = false;
```

Comment:
this is getting larger, should we create a state mapper?

Reply:
A mapper added